addons/web/static/src/js/views/kanban/kanban_quick_create.js

```diff
@@ -116,17 +116,23 @@ var RecordQuickCreate = AbstractQuickCreate.extend({
     template: "KanbanView.QuickCreate",
     /**
      * @override
+     * @param {Widget} parent
+     * @param {Object} options
+     * @param {string} options.width defines the element's width
+     * @param {string} options.defaultName the record's default name
```

Comment:
+     * @param {string} [options.defaultName] the record's default name

to mark a parameter as optional, it should be enclosed in brackets